MovieTime/MovieTime/Core/Router/MTRouter.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  MTRouter.swift
+//  MovieTime
+//
+//  Created by Enrique Melgarejo on 10/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import UIKit
+
+enum MTAppState {
+    case splashScreen
+    case feed
+}
+
+final class MTRouter: NSObject {
```

Comment:
why does this class inherit from NSObject?

Reply:
I started developing the `MTRouter` thinking I would use some third party library. And one that I've already implemented is still being developed in Objc, so to use my router in the Objc world, I would probably need to inherit from *NSObject*.
But to use a third party library would be like using a bazooka to kill an ant, since it is only being used to transpose the initial screens.
In the end, I forgot to remove the *NSObject*.